### PR TITLE
Adding Go Flashcards

### PR DESCRIPTION
**Pull request by roymckenzie**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -15675,6 +15675,31 @@
       ],
       "date_added": "Feb 12 2018",
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Go Flashcards",
+      "category-ids": [
+        "education"
+      ],
+      "description": "Go Flashcards for iOS and WatchOS allows users to create stacks of flashcards and review them on their iPad, iPhone, or Apple Watch. All stacks and stack progress sync over CloudKit.",
```
Comment (vpeschenkov) [inline contents.json]: Could you remove `iOS` from the description, according to the point `Avoid iOS or open-source in description as it is assumed`?

**Comment by vpeschenkov** [comment]
@roymckenzie thank you!

**Comment by roymckenzie** [comment]
Thanks @vpeschenkov. I forgot to add the url to the app in the App Store. I didn't see that in the (CONTRIBUTING.md)[https://github.com/dkhamsing/open-source-ios-apps/blob/master/.github/CONTRIBUTING.md]. Should I make another PR for that?

**Comment by vpeschenkov** [comment]
@roymckenzie Yes, make another PR for that, please.

**Pull request merged**